src/main/kotlin/org/rust/cargo/project/model/impl/CargoTomlWatcher.kt

```diff
@@ -85,6 +85,7 @@ class CargoTomlWatcher(
                     }
                 }
                 event is VFileContentChangeEvent -> false
+                event.pathEndsWith("bazel-bin") -> true
```

Comment:
When running `bazel build` for the first time, no file change event is raised for the `Cargo.toml` file inside `bazel-bin` - we only get a folder creation event for `bazel-bin` itself. So we listen for it here.